src/Extension/DateTimeExtension.php

```diff
@@ -0,0 +1,240 @@
+<?php
+
+namespace Faker\Core\Extension;
+
+/**
+ * FakerPHP extension for Date-related randomization.
+ *
+ * Functions accepting a date string use the `strtotime()` function internally.
+ *
+ * @experimental
+ *
+ * @since 1.20.0
+ */
+interface DateTimeExtension extends Extension
+{
+    /**
+     * Get a DateTime object between January 1, 1970, and `$until` (defaults to "now").
+     *
+     * @param \DateTime|int|string $until    maximum timestamp, defaults to "now"
+     * @param string|null          $timezone zone timezone for generated date, fallback to `DateTime::$defaultTimezone` and `date_default_timezone_get()`.
+     *
+     * @see \DateTimeZone
+     * @see http://php.net/manual/en/timezones.php
+     * @see http://php.net/manual/en/function.date-default-timezone-get.php
+     *
+     * @example DateTime('2005-08-16 20:39:21')
+     */
+    public function dateTime($until = 'now', string $timezone = null): \DateTime;
```

Comment:
```suggestion
    public function dateTime(string $until = 'now', string $timezone = null): \DateTime;
```

Reply:
It can be int, string, and some others too.